### ISSUE TITLE
Progress bar nodes have accessible name/must conform to valid values

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#23775]
## Issue Description

The Blue button progress bar spinner implementation on VAMC pages has been found to cause two new AXE errors:  

- ARIA progressbar nodes must have an accessible name
- ARIA attributes must conform to valid values

Previously the spinner did not trip during testing. Likely the HTML markup needs to be updated or the component needs to be updated to align with the design system. 

<details>
<summary> Screenshots </summary>

![image](https://user-images.githubusercontent.com/55411834/116124447-8b1f2c00-a681-11eb-9120-5a6dbbd28897.png)

![image](https://user-images.githubusercontent.com/55411834/116125506-cf5efc00-a682-11eb-9ea9-38d2b5c427f7.png)

![image](https://user-images.githubusercontent.com/55411834/116124690-d8030280-a681-11eb-98e4-83f03412d5a1.png)

</details>

This issue is present on the following example pages (it occurs on multiple health care system pages - not just Sioux Falls)  

1. [Make an appointment](https://www.va.gov/sioux-falls-health-care/make-an-appointment/)
2. [Pharmacy](https://www.va.gov/sioux-falls-health-care/pharmacy/)

---
## Tasks
- [ ] Update spinner to align with [Design system](https://design.va.gov/storybook/?path=/docs/components-loadingindicator--default)

## Acceptance Criteria
- [ ] The Blue button spinner on VAMC _Make an appointment_ pages does not throw an AXE violation 
- [ ] The Blue button spinner on VAMC _Pharmacy_ pages does not throw an AXE violation

**Comment by mmiddaugh** [comment]
The progress bar defect on Medical records pages should be resolved by PW #21787

**Comment by mmiddaugh** [comment]
We think the issue is resolved as a part of Public Website's PU: Update aria-label for progressbar to have a default value
#16956

@1Copenut - can you please confirm?

**Comment by 1Copenut** [comment]
@mmiddaugh I'll add this to my list of items to confirm. Thank you!

**Comment by 1Copenut** [comment]
@mmiddaugh I ran this on the Sioux Falls links for Make an appointment and Pharmacy listed in the description. Saw no axe violations for the loading indicator so I think we've got this one fixed. I'll keep an eye on it in the CI logs, moving the issue to closed.
